src/main/kotlin/no/nav/familie/ef/søknad/api/dto/Kvittering.kt

```diff
@@ -1,3 +1,5 @@
 package no.nav.familie.ef.søknad.api.dto
 
-data class Kvittering(val text: String)
\ No newline at end of file
+import java.time.LocalDateTime
+
+data class Kvittering(val text: String, val motattDato: LocalDateTime?)
```

Comment:
```suggestion
data class Kvittering(val text: String, val mottattDato: LocalDateTime?)
```

Trenger vi `?` her?

Reply:
La til denne for å være kompatibel med søknads controller - sender kvittering også ved catching av feil. (Altså vi returnerer 200, men med feilmelding). Da hadde jeg ikke lyst til å finne på en dato ;)  

Har lyst til å endre i søknadskontroller - ikke returnere kvittering ved feil.

Reply:
Bra du har oversikten!